### DISCUSSION TITLE
Cápsula parte 2 sin sonido

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#76] [Tarea 1] [answered]
Hola, me parece que la cápsula para la parte 2 no tiene sonido. Al menos, no escucho nada en mi dispositivo, pero sí escucho el audio de la cápsula del primer video.

**Comment by 4n1t4** [comment, marked as answer]
Holi!
Si, tuvimos problemas con el audio de la cápsulas.  Sin embargo ya fue solucionado. El link para capsula parte 2 es el siguiente: https://www.youtube.com/watch?v=pSt56uG2uMQ

**Reply by MaxAl100** [reply]
okis, muchas gracias! 😄
